src/Sulu/Bundle/SecurityBundle/Factory/UserFactory.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
We haven't used strict_types yet in the code base, so I would like to not start with that yet.